### PR TITLE
docs(sphinx-conf): Fix CLI documentation process

### PR DESCRIPTION
**Pull request by santiagogaray**
Hi @chriswmackey,

I found a unique situation in this repo that I would like to make sure I handle it correctly.
The [ladybug_comfort/cli/__init__.py](url) file has a ``comfort`` group without any commands added. 
```
# command group for all comfort extension commands.
@click.group(help='ladybug comfort commands.')
@click.version_option()
def comfort():
    pass
```
Currently I ignore this scenarios and don't' create a ``main`` section without commands, but I realize now that we have the ``@click.version_option`` decorator that will probably render something like the top portion of the honeybee-energy cli doc...
Do we want to have this standalone option in the main section? My guess is yes but I wanted to double check!

image

### CODE REVIEW

**Review by chriswmackey — APPROVED** [review]
Hey @santiagogaray .

Sorry for being late here. I think it is nice to show that people can check the version even when there are no other commands in the main group. But it's not critical and I would be fine merging this PR as is. Let me know if you would like me to just merge this or if you would want to add support for the version command docs before we merge.

**Comment by santiagogaray** [comment]
Sounds good @chriswmackey. Feel free to merge this one at it is.
Will add support for the version option in a future version.

**Pull request merged**
